hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/utils/Pipelines.java

```diff
@@ -137,7 +137,7 @@ public static DataStreamSink<Object> bulkInsert(Configuration conf, RowType rowT
         SortOperatorGen sortOperatorGen = new SortOperatorGen(rowType, partitionFields);
         // sort by partition keys
         dataStream = dataStream
-            .transform("partition_key_sorter",
+            .transform("partition_key_sorter" + ":" + conf.getString(FlinkOptions.TABLE_NAME),
                 TypeInformation.of(RowData.class),
```

Comment:
Maybe only the write op with table name is enough, for e.g 
- bucket_bulk_insert
- hoodie_bulk_insert_write
- hoodie_append_write
- bucket_write
- stream_write

And we can extract a common util method here like

`writeOpIdentifier(String, Configuration)` 

for generating operator name with table suffix.

Reply:
right , i will try to amend it with your suggestion

Reply:
i have create a new PR in https://github.com/apache/hudi/pull/5744